File: public/code/hello-world-problem.py
# Import the Jutge API client
from jutge_api_client import JutgeApiClient

# Create a Jutge API client
jutge = JutgeApiClient()

# Get "Hello, world!" problem (P68688); print title and author 
problem = jutge.problems.get_problem("P68688_en")
print(problem.title)
print(problem.abstract_problem.author)

# print the English statement
statement = jutge.problems.get_text_statement("P68688_en")
print(statement)

File: public/code/hello-world-problem.ts
// Import the Jutge API client and its extras
import { JutgeApiClient } from "./jutge_api_client"

// Make a client
const jutge = new JutgeApiClient()

// Get P68688 (Hello, world!) problem and print its title and author.
const problem = await jutge.problems.getProblem("P68688_en")
console.log(problem.title)
console.log(problem.abstract_problem.author)

const statement = await jutge.problems.getTextStatement("P68688_en")
console.log(statement)
